packages/jazz-browser-auth-clerk/src/index.ts

```diff
@@ -24,6 +31,33 @@ export class BrowserClerkAuth implements AuthMethod {
 
     async start(): Promise<AuthResult> {
         if (this.clerkClient.user) {
+            // Check local storage for credentials
```

Comment:
I haven't tried this, but in case the Clerk cookie expires / we reload, do we even have a user in the clerkClient? I.e. do we need to check local storage even above this `if`?

Reply:
Fair point, I'll test that scenario

Reply:
Fixed